SU2_DOT/include/SU2_DOT.hpp

```diff
@@ -32,14 +32,16 @@
 #include "../../Common/include/mpi_structure.hpp"
 #include "../../Common/include/omp_structure.hpp"
 
+#define ENABLE_MAPS
+#include "../../Common/include/config_structure.hpp"
+
```

Comment:
Maybe putting a `#undef ENABLE_MAPS` after the include?

Reply:
Done